foundations/installations/installing_node.md

```diff
@@ -43,13 +47,41 @@ command -v nvm
 
 if this returns `nvm: command not found` close the terminal and re-open it.
 
-On OS X you may continue to see the same message. If this happens, open your .bash_profile and add the following line of code:
+In macOS 10.15 and above, the default shell is zsh and nvm will look for .zshrc to update, none is installed by default.
+
+Simply run the following commands:
+
+~~~bash
+touch ~/.zshrc
+~~~
+
+~~~bash
+curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.3/install.sh | bash
+~~~
+
+
```

Comment:
We should move this instruction up above the manual `export ...` commands as it should be handled during installation.

```suggestion
Once the installation is finished, open a new terminal window and test your nvm installation by running:

~~~bash
nvm --version
~~~
```